themes/default/content/blog/oidc-with-azure/index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Announcing OIDC Support for Pulumi Azure Providers"
+
+# The date represents the post's publish date,
+# and by default corresponds with the date this file was generated.
+# Posts with future dates are visible in development,
+# but excluded from production builds.
+# Use the time and timezone-offset portions of of this value
+# to schedule posts for publishing later.
+date: 2023-05-17T12:36:04-07:00
+
+# Use the meta_desc property to provide a brief summary
+# (one or two sentences) of the content of the post,
+# which is useful for targeting search results or social-media previews.
+# This field is required or the build will fail the linter test.
+# Max length is 160 characters.
+meta_desc: A guide to using OIDC authentication with Azure in Pulumi
+
+# The meta_image appears in social-media previews and on the blog home page.
+# A placeholder image representing the recommended format, dimensions and aspect ratio
+# has been provided for you.
+meta_image: meta.png
+
+# At least one author is required.
+# The values in this list correspond with the `id` properties
+# of the team member files at /data/team/team.
+# Create a file for yourself if you don't already have one.
+authors:
+    - thomas-kappler
+
+# At least one tag is required.
+# Lowercase, hyphen-delimited is recommended.
+tags:
+    - azure
+
+---
+
+We are happy to announce the delivery of Azure OIDC authentication, one of the [most requested features](https://github.com/pulumi/pulumi-azure-native/issues/1324) for the [Pulumi Azure Native Provider](https://www.pulumi.com/registry/packages/azure-native/). With the v1.100.0 release, OpenID Connect (OIDC) authentication is now fully supported in both the native and [classic](https://www.pulumi.com/registry/packages/azure/) providers. Let’s dig in to learn what it is, how it works, and why it’s useful.
+
+<!--more-->
+
+{{% notes type="info" %}}
+[Pulumi announced OIDC support for Deployments](https://www.pulumi.com/blog/oidc-blog/) in January. This article is about the same OIDC protocol, this time used to authenticate your Pulumi program to access Azure resources.
```

Comment:
If I'm understanding correctly, the key difference here from the earlier blog post is this covers use cases outside Deployments (like your example of using GitHub Workflows). Is that correct? If so, perhaps rewording this section slightly to make that distinction would be helpful for readers.

Reply:
I gave it a shot, let me know what you think